test-swift-rust-integration.sh

```diff
@@ -25,3 +25,12 @@ xcodebuild \
   -project SwiftRustIntegrationTestRunner.xcodeproj \
   -scheme SwiftRustIntegrationTestRunner \
   clean test
+
+if ! sh ./test-release-fails/build.sh; then
+    echo "Build failed as expected"
+else
+    echo "Error: Build succeeded but was expected to fail"
+    exit 1
+fi
```

Comment:
We don't need to maintain two cases.

When I said to make a failing cast I just meant to 

1. Make it fail
2. Change `swift-bridge`
3. Confirm that it now passes

So that we were sure that our changes actually worked and that we didn't:

1. Make a test that already passed (BAD)
2. Make swift-bridge changes
3. Confirm that test passes (BAD, it already passed to begin with. It was a bad test)

Does that make sense?